cpp/dolfinx/fem/interpolate.h

```diff
@@ -952,7 +936,70 @@ void interpolate(Function<T>& u, const Function<T>& v,
   {
     // Get mesh and check that functions share the same mesh
     if (auto mesh_v = v.function_space()->mesh(); mesh != mesh_v)
-      impl::interpolate_nonmatching_meshes(u, v, cells);
+    {
```

Comment:
Can we keep the function `impl::interpolate_nonmatching_meshes` by redefining it to take the precomputed data?

Reply:
I didn't mean to remove it, I just forgot to restore it :)